### DISCUSSION TITLE
APISIX manage front-end pages

### DISCUSSION THREAD

**Discussion by daemonmanx** [apache/apisix#6082] [Q&A]
Nginx deployment front-end pages step

1. Vue development front-end application

2. Generate static files to `/data/html`

```shell
npm run build
```

3. nginx config

```shell
server {
  listen <*>;
  server_name <*>;
  
  root /data/html;
  index index.html;
}
```

## Question

1. Can APISIX manage front-end pages?
2. Can all pages be synchronized when APISIX is clustered?

**Comment by juzhiyuan** [comment]
Hi @daemonmanx, actually you want to know if Apache APISIX could proxy static files, yes it could, but we don't recommend using it :) see https://github.com/apache/apisix/issues/4889#issuecomment-905161689

**Reply by daemonmanx** [reply]
Is still managed by nginx, but not hosted by APISIX. What I mean is that through APISIX dynamic configuration, the page of each node updated synchronously
